src/pages/fund-wallet/FundWalletTabs/index.tsx

```diff
@@ -7,9 +7,9 @@ interface FundWalletTabsProps {
 }
 
 export function FundWalletTabs({ activeTab, setActiveTab }: FundWalletTabsProps) {
-  const tabItems = [
-    { index: FundWalletTab.Buy, label: 'Buy' },
-    { index: FundWalletTab.Exchange, label: 'Exchange' },
+  const tabItems: { index: FundWalletTab; children: JSX.Element }[] = [
```

Comment:
Nit: Maybe extract `{ index: FundWalletTab; children: JSX.Element }` to a new type to improve readability.